File: 1009.py
#1009 SALÁRIO COM BÔNUS
nome = input()
salarioFixo = float(input())
totalVendas = float(input())

comissão = totalVendas*0.15
total = salarioFixo+comissão
print("TOTAL = R$ {:.2f}" .format(total))

File: javascript/1009.js
/*Salário com Bônus*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

const lines = input.split("\n");

const Nome = lines.shift();
const SalarioFixo = parseFloat(lines.shift());
const Vendas = parseFloat(lines.shift());

const Valor = Vendas * 0.15;

const ValorF = SalarioFixo + Valor;

console.log("TOTAL = R$ " + ValorF.toFixed(2));